File: Selenium/Project/Python_Project/Activity2.py
from selenium import webdriver

# Initialize the WebDriver
driver = webdriver.Chrome()

try:
    # Navigate to the website
    driver.get('https://alchemy.hguy.co/jobs/')
    
    # Get the heading of the webpage
    heading = driver.find_element("xpath","//h1[text()='Welcome to Alchemy Jobs']").text
    
    # Verify the heading
    assert heading == "Welcome to Alchemy Jobs", "Website heading does not match"
    print("Website heading verification successful.")

finally:
    # Close the browser
    driver.quit()

File: Selenium/Project/Project_Java/Activity2.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        // Navigate to the website
        driver.get("https://alchemy.hguy.co/jobs/");

        // Get the heading of the webpage
        String heading = driver.findElement(By.xpath("//h1[text()='Welcome to Alchemy Jobs']")).getText();

        // Verify the website heading
        if (heading.equals("Welcome to Alchemy Jobs")) {
            System.out.println("Website heading matches: " + heading);
        } else {
            System.out.println("Website heading does not match: " + heading);
        }

        // Close the browser
        driver.quit();
    }
}
